federation/pkg/federation-controller/secret/secret_controller_test.go

```diff
@@ -34,10 +34,27 @@ import (
 	"k8s.io/kubernetes/pkg/types"
 	"k8s.io/kubernetes/pkg/util/wait"
 
+	"github.com/golang/glog"
 	"github.com/stretchr/testify/assert"
 )
 
+const (
+	testIterations = 1
+)
+
 func TestSecretController(t *testing.T) {
+	for i := 0; i < testIterations; i++ {
```

Comment:
Why do this at all? You could instead use [`cmd/stress`](https://godoc.org/golang.org/x/tools/cmd/stress) to accomplish this

```
$ go get -u  golang.org/x/tools/cmd/stress
$ go test -c pkg.test
$ stress ./pkg.test -test.run=TestMyFunc
```